requirements.txt

```diff
@@ -5,6 +5,7 @@
 #    pip-compile --output-file=requirements.txt requirements.in
 #
 click==7.0                # via flask
+codecov==2.0.15
```

Comment:
@miteshvp you will need a codecov token to be created for this repo https://github.com/fabric8-analytics/metrics-accumulator/blob/master/runtests.sh#L70. @tisnik can help you get one if you have not done yet.

Reply:
@samuzzal-choudhury we already have the token and the codecov is setup for this repository https://codecov.io/gh/fabric8-analytics/metrics-accumulator